accel/tcg/log_instr.c

```diff
@@ -739,18 +740,26 @@ static void do_global_loglevel_switch(CPUState *cpu, run_on_cpu_data data)
  * the current TB is finished. If we clear the global flag immediately
  * we will end up emitting stale instructions.
  */
-void qemu_log_instr_global_switch(bool request_stop)
+int qemu_log_instr_global_switch(int log_flags)
 {
     CPUState *cpu;
     qemu_log_instr_loglevel_t level;
 
-    level = (request_stop) ? QEMU_LOG_INSTR_LOGLEVEL_NONE :
-        QEMU_LOG_INSTR_LOGLEVEL_ALL;
+    if(log_flags & CPU_LOG_INSTR_U) {
```

Comment:
Needs some clang-format love for the series

Reply:
Yeah and ideally checkpatch.pl too (see #169)